### PR TITLE
Added FHV and FAQ new rules

### PR DESCRIPTION
**Pull request by drawntotheflame**
Adding FHV rulings and FAQ rulings.

### CODE REVIEW

File: src/AppBundle/Resources/views/Default/rulesreference.html.twig [hunk 1]
```diff
@@ -93,13 +97,15 @@
...
 		</ul>
+	<li><a href="#Customizable">Customizable</a></li>
 	<li><a href="#Dealing_Damage_Horror">Dealing Damage/Horror</a></li>
 	<li><a href="#Deck">Deck</a></li>
 	<li><a href="#Deckbuilding">Deckbuilding</a></li>
 		<ul>
 		<li><a href="#Classes">Classes</a></li>
 		<li><a href="#Deckbuilding_Options">Deckbuilding Options</a></li>
 		</ul>
+	<li><a href="#Day_and_Night">Day and Night</a></li>
```
Comment (drawntotheflame) [inline src/AppBundle/Resources/views/Default/rulesreference.html.twig]:
```suggestion
```

File: src/AppBundle/Resources/views/Default/rulesreference.html.twig [hunk 2]
```diff
@@ -93,13 +97,15 @@
 		<ul>
 		<li><a href="#Ignoring_All_Costs">"Ignoring all costs"</a></li>
 		</ul>
+	<li><a href="#Customizable">Customizable</a></li>
 	<li><a href="#Dealing_Damage_Horror">Dealing Damage/Horror</a></li>
```
Comment (drawntotheflame) [inline src/AppBundle/Resources/views/Default/rulesreference.html.twig:102]:
```suggestion
	<li><a href="#Day_and_Night">Day and Night</a></li>
	<li><a href="#Dealing_Damage_Horror">Dealing Damage/Horror</a></li>

```

**Comment by drawntotheflame** [comment]
@Kamalisk this can go live when you have a moment - thanks.

**Comment by Chr1Z93** [comment]
Looks good! Looking forward to getting this live to easier reference these new keywords 😊

**Pull request merged**